pom.xml

```diff
@@ -45,26 +45,26 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>4.2.5</version>
+        <version>4.2.6</version>
```

Comment:
Can you add a temporary jackson-databind fix?
https://github.com/folio-org/raml-module-builder/pull/1036/files

Reply:
I tried that.. To no effect.. I did mvn dependency:tree on new mod-configuration and still 2.13.1 was there.

Reply:
Good catch, thanks!

The jackson-databind entry needs to go before the jackson-bom entry in pom.xml: https://github.com/folio-org/raml-module-builder/pull/1038